src/UserForm.js

```diff
@@ -292,25 +278,44 @@ class UserForm extends React.Component {
     submitter();
   }
 
-  getActionMenu = ({ onToggle }) => {
-    const { onCancel } = this.props;
-    const handleClick = () => {
-      onCancel();
-      onToggle();
-    };
+  getPaneFooter() {
+    const {
+      id,
+      pristine,
+      submitting,
+      invalid,
+      onCancel,
+      buttonStyle,
+      cancelButtonStyle,
+    } = this.props;
+
+    const disabled = pristine || submitting || invalid;
+    const testId = id || uniqueId('edituser-');
+    const cancelLabel = this.props.cancelLabel || <FormattedMessage id="ui-users.cancel" />;
```

Comment:
I suggest removing `this.props.cancelLabel` prop at all as if labels will need to be changed, there will be only change for `FormattedMessage`.